### PR TITLE
Logout button on Nav

### PR DESCRIPTION
**Pull request by asitu**
:tickets: **Ticket(s)**: Closes #

---

## :construction_worker: Changes
Logout button on the nav (as a component) without global styling

### CODE REVIEW

File: daemon/web/components/buttons/LogoutButton/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../index.sass';
+
+const LogoutButton = ({ style }) => (
```
Comment (bobheadxi) [inline daemon/web/components/buttons/LogoutButton/index.js]: LogoutButton should always look the same, right? I think we can omit the `style` prop

File: daemon/web/components/buttons/ShutdownButton/index.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+const ShutdownButton = () => (
   <div>
-    <button className="button" type="button" style={style}>
+    <button className="button" type="button">
 Shut Down Containers
     </button>
   </div>
 );
 
 ShutdownButton.propTypes = {
```
Comment (bobheadxi) [inline daemon/web/components/buttons/ShutdownButton/index.js]: since there are no props, this can be removed entirely :)

Comment (bobheadxi) [reply daemon/web/components/buttons/ShutdownButton/index.js]: WAIT i think you changed the wrong button? this is `ShutdownButton`

Comment (bobheadxi) [reply daemon/web/components/buttons/ShutdownButton/index.js]: should probably be updating `LogoutButton` instead

Comment (asitu) [reply daemon/web/components/buttons/ShutdownButton/index.js]: Okay lol yep.. Should it be removed for all the buttons? all the buttons now don't follow the global style classes

Comment (bobheadxi) [reply daemon/web/components/buttons/ShutdownButton/index.js]: I think we should leave it for the other buttons for now, until we can check if they use it or not.

The buttons as a whole probably need a rework - we probably don't want `ShutdownButton` and `LogoutButton`, but a `PrimaryButton` and `SecondaryButton` instead, to improve re-usability - but this is fine for the time being.

Comment (bobheadxi) [reply daemon/web/components/buttons/ShutdownButton/index.js]: So yeah, just remove it for `LogoutButton`, and leave it as is for the others 🚀

Comment (asitu) [reply daemon/web/components/buttons/ShutdownButton/index.js]: OKayy 👍

**Pull request merged**